### ISSUE TITLE
Continuous question on #157

### ISSUE THREAD

**Issue by v1v2v3vv55** [SupremeTechnopriest/react-idle-timer#158]
Hi thanks for answering on that one, it is quick and I really appreciate it! But I still feel unclear since I don't call reset() when user is idling:

When I testing my app the timer will be reset when there's a mouse movement/click or input from keyboard, and these actions trigger onActive, which triggered the reset(), to clarify, can I prevent the situation that automatically calling onActive or reset()? Because I'm trying to synchronize the behavior of the timer.

**Comment by SupremeTechnopriest** [comment]
Yeah you can set `stopOnIdle` to `true` and when the user becomes Idle, you will have to manually call `reset` or `start` to start the timer again.  Does this help?

**Comment by v1v2v3vv55** [comment]
Hi thanks! But if I set `stopOnIdle` to` true`, the timer will be paused right? For my app I do want the timer continue counting down when user is idling but not reset timer if user simply moving the mouse and pressing a single button on the keyboard. Which means the timer will reset only when `startOnMount` being called.

**Comment by SupremeTechnopriest** [comment]
The timer will count down with `stopOnIdle` set, it just wont reset on user activity once idle.  You would have to call `start` or `reset` manually to restart the timer.  Im pretty sure this is what you want.

**Comment by v1v2v3vv55** [comment]
Thanks a lot!

**Comment by v1v2v3vv55** [comment]
Hi sorry for reopening this issue, I have set `stopOnIdle` to `true`, however the timer will still get reset once I move my mouse for a little bit without doing left/right click regardless of idling or not, since I hope the timer to reset only when users doing mouse click and pressing key , what I tried is changing the default events like this in order to prevent the listener retrieve some events that I don't wish it to be listened:
`<IdleTimer
          ref={ref => { this.idleTimer = ref }}
          element={document}
          onActive={this.onActive}
          onIdle={this.onIdle}
          onAction={this.onAction}
          debounce={250}
          timeout={this.state.timeout} 
          events={[mousedown,keydown]}/>`
Which is doesn't work and showing me the error that mousedown and keydown is not defined. By common sense since the default includes not only these 2 events so I think it is defined by logic. When I change it to mouseDown and onKeydown according to the quick fix it showing the overload error as below:
No overload matches this call.
  Overload 2 of 2, '(props: IdleTimerProps, context: any): IdleTimer', gave the following error.
    Type '(props: any) => any' is not assignable to type 'EVENTS'.
  Overload 2 of 2, '(props: IdleTimerProps, context: any): IdleTimer', gave the following error.
    Type '(this: Window, ev: KeyboardEvent) => any' is not assignable to type 'EVENTS'.
What should I do in this situation? Thanks again!
